util/src/main/java/org/datacommons/util/TmcfCsvParser.java

```diff
@@ -191,24 +235,58 @@ public void process(CSVRecord dataRow) {
           instanceValues.addTypedValues(newTypedValue.build());
         } else if (typedValue.getType() == Mcf.ValueType.TABLE_COLUMN) {
           // Replace column-name with cell-value
-          int columnIndex = toColumnIndex(typedValue.getValue());
-          assert columnIndex < dataRow.size() : "Unexpected inconsistent row " + dataRow.toString();
+          McfParser.SchemaTerm term = McfParser.parseSchemaTerm(typedValue.getValue(), errCb);
+          if (term == null) {
+            continue;
+          }
+          if (term.type != McfParser.SchemaTerm.Type.COLUMN) {
+            addLog(
+                Debug.Log.Level.LEVEL_ERROR,
+                "CSV_UnexpectedNonColumn",
+                "Unexpected non-column "
+                    + typedValue.getValue()
+                    + " for property "
+                    + currentProp
+                    + " in TMCF entity "
+                    + templateEntity);
+            continue;
+          }
+          if (!cleanedColumnMap.containsKey(term.value)) {
+            addLog(
+                Debug.Log.Level.LEVEL_ERROR,
+                "CSV_MissingTmcfColumn",
+                "Missing column " + term.value);
```

Comment:
clarify that the error is in the tmcf? and will we be able to specify which tmcf file was used?

Reply:
clarified.  for *every* addLog() call, the file name and line name gets added internally.